src/GitVersionCore/BranchConfigurationCalculator.cs

```diff
@@ -37,9 +37,6 @@ public class BranchConfigurationCalculator
 
                 return keyValuePair;
             }
-
-            const string format = "Multiple branch configurations match the current branch branchName of '{0}'. Matching configurations: '{1}'";
-            throw new Exception(string.Format(format, currentBranch.FriendlyName, string.Join(", ", matchingBranches.Select(b => b.Key))));
```

Comment:
It would probably be nice to know that there were multiple matches, even though it now doesn't fail.